python/makeALPACAEvents.py

```diff
@@ -34,7 +32,7 @@ def inputWrite(mres,gax,nev,Lmin,Lmax):#need to apply Rmin & Rmax as well..
     L   = ALPACAFormatting(float(Lmax)-float(Lmin))
     Lmin    = ALPACAFormatting(Lmin)
     Lmax    = ALPACAFormatting(Lmax)
-    with open("input.DAT","w") as f:
+    with open("input_"+str(mres)+"_"+str(gax)+".DAT","w") as f:
```

Comment:
I would evaluate this name once and then reuse it everywhere in this file. I see at least 3 instances of this evaluation.
If you ever want to change it in future, it's easy to make mistakes by forgetting to change it somewhere (or introducing a typo in one of the 3 cases).

Same applies for the output names.

This is something you can have in the global scope of the file (or maybe in the module namespace).

Reply:
I made a  defn for this situation.